tools/releaser/scripts/update-version.sh

```diff
@@ -19,7 +19,7 @@ echo "Modifying $SDK_MAJOR_VERSION to $BUMPED_MAJOR_VERSION Resource Version acr
 npm exec -c "replace-in-file /$SDK_MAJOR_VERSION/g $BUMPED_MAJOR_VERSION $VERSION_UPDATE_PATHS --isRegex"
 
 echo "Creating empty breaking changes file for $BUMPED_MAJOR_VERSION"
-echo "# Breaking Changes" > "$script_path/../breaking_changes/${BUMPED_MAJOR_VERSION}.md"
+printf "\n# Breaking Changes\n" > "$script_path/../breaking_changes/${BUMPED_MAJOR_VERSION}.md"
```

Comment:
```suggestion
echo "# Breaking Changes\n" > "$script_path/../breaking_changes/${BUMPED_MAJOR_VERSION}.md"
```

Reply:
shellcheck suggests printf :(

Reply:
oh well, can you then just remove the opening `\n` I don't think is necessary

Reply:
https://github.com/mongodb/atlas-sdk-go/releases/tag/v20230201008.0.0 seems to be missing `\n` at the beginning for some reason

Reply:
that seems to be a problem of the releaser not of this script

Reply:
Closing PR then. Sorry for confusion